tests/lib/rook/base.py

```diff
@@ -72,9 +77,16 @@ def install(self):
 
         # TODO(jhesketh): Check if sleeping is necessary
         time.sleep(10)
-
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'cluster.yaml'))
+    logger.info("Rook successfully installed and ready!")
+
+    def install_ceph(self, cluster_yaml=None, health_test=True):
```

Comment:
Is there are way to actually use different method params? I wonder if we really want `cluster_yaml` and `health_test` as parameters here

Reply:
How to install a ceph cluster with a different cluster.yaml?

Reply:
I missed the usage of the parameters. sorry for the noise.